app/assets/stylesheets/components/_navbar.scss

```
@@ -0,0 +1,18 @@
+.navbar-dev {
+  &:before {
+    @extend .progress-bar-striped;
+    @extend .bg-warning;
+
+    content: "";
+    position: absolute;
+    bottom: 0;
+    height: 4px;
+    width: 100%;
+    background-size: 4px 4px;
```

tiniest stripes I've ever seen

```suggestion
    background-size: 40px 40px;
```